### DISCUSSION TITLE
Just want to say Thank YOU!!!!!

### DISCUSSION THREAD

**Discussion by tanthammar** [kiliman/tailwindui-crawler#69] [General]
Saves a lot of time. Thank you for making this package.

**Comment by connecteev** [comment]
Couldn't agree more - amazing work on this repo!

**Comment by kiliman** [comment]
@connecteev thanks for all the feedback. I'm so glad it's been useful to you.

**Comment by connecteev** [comment]
@kiliman thanks for the encouragement - really, all credit goes to you for the work you've done and I appreciate you listening for feedback.

**Comment by Julianwan** [comment]
Also want to say thank you :)

Is there a method for pulling the 'free' components if we don't have a Tailwind UI login?

**Comment by kiliman** [comment]
@Julianwan thanks!

Funny you should ask. The initial version of the crawler didn't verify if you actually logged in properly and it would start crawling the default page which had the "free" components. I added the login check in #3.

I supposed I could make it an option. But you can also just remove the login check from index.js. See https://github.com/kiliman/tailwindui-crawler/commit/618de6e37c6c33962ad90cea2fea432d243b5969

**Comment by Julianwan** [comment]
Nice! Thank you @kiliman ! That's awesome!

**Comment by cinesoftcentrocuba** [comment]
Thank you @kiliman !!!
